src/vnsw/agent/oper/physical_interface.cc

```diff
@@ -130,13 +130,12 @@ void PhysicalInterface::PostAdd() {
         close(fd);
         return;
     }
-#endif
 
-#ifdef _WIN32
-    closesocket(fd);
-#else
     close(fd);
+#else
+    closesocket(fd);
```

Comment:
I don't like this change. Probably Hari told you to do this but it's counterintuitive. There is a comment `// Not yet suported on Windows` before `#ifndef _WIN32` (for enabling promiscuous mode). Now it looks like closing the socket is not supported on Windows. I thought about doing this exact change during cleanup but decided that it should be two separate ifdefs - first for code that is not yet supported for Windows and second for closing the socket.